src/Domain.jl

```diff
@@ -1599,6 +1599,49 @@ function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=
     return jac
 end
 
+function jacobiany!(jac::Q,y::U,p::W,t::Z,domain::D,interfaces::Q3,colorvec::Q2=nothing) where {Q3<:AbstractArray,Q2,Q<:AbstractArray,U<:AbstractArray,W,Z<:Real,D<:ConstantVDomain}
+    ns,cs,T,P,V,C,N,mu,kfs,krevs,Hs,Us,Gs,diffs,Cvave = calcthermo(domain,y,t,p)
+    jacobiany_ns!(jac,y,p,t,domain,[],cs,kfs,krevs,nothing)
+    
+    cpdivR,hdivRT1,sdivR = calcHSCpdless(domain.phase.vecthermo,T)
+    
+    dydt = zeros(size(y))
+    dydtreactor!(dydt,y,t,domain,interfaces,;p=p,sensitivity=false)
+    
+    dCvavedn = zeros(domain.indexes[2]-domain.indexes[1]+1)
```

Comment:
It'd be nice if we could avoid allocating this vector. I think it might be okay to just recalculate dCvavedn as needed. It's not that expensive to recalculate and in most cases I expect this won't have an outlet and it won't matter at all.